db/fixtures/miq_product_features.yml

```diff
@@ -4776,6 +4776,10 @@
         :description: Ansible Playbooks Refresh
         :feature_type: control
         :identifier: embedded_configuration_script_payload_refresh
+      - :name: Edit Tags
+        :description: Edit Tags for the selected Ansible Playbooks
+        :feature_type: control
+        :identifier: ansible_playbook_tag
```

Comment:
@hstastna should this feature also be named as all other features in this section for consistency `embedded_configuration_script_payload_xxx`